src/main/java/de/tum/in/www1/artemis/service/notifications/MailService.java

```diff
@@ -443,7 +400,6 @@ public void sendWeeklySummaryEmail(User user, Set<Exercise> exercises) {
         context.setVariable(BASE_URL, artemisServerUrl);
 
         String content = templateEngine.process("mail/weeklySummary", context);
-
-        sendEmail(user, subject, content, false, true);
+        mailSendingService.sendEmail(user, subject, content, false, true);
```

Comment:
Now we have an asynchronous method calling another (now actually) asynchronous method as the last call.
Can we maybe get rid of one of the asyncs?

Reply:
Great catch, Thanks!